app/views/content/funding-and-support/tuition-fee-and-maintenance-loans.md

```diff
@@ -58,10 +69,17 @@ The maximum amount you may be eligible to borrow is:
 * $finance_maintenanceloan_notwithparentsinsidelondon$, if you live away from your parents, in London
 * $finance_maintenanceloan_oversixty$, if you're 60 years or over on the first day of the academic year of your course
 
-
 $student-finance-calculator$
 
-## If you come from outside England
+## Check your eligibility for a bursary or scholarship
+
+You may also be eligible for a tax-free bursary or scholarship, depending on the subject you're training to teach.
+
+You do not have to pay a bursary or scholarship back. You can get a bursary or scholarship as well as any tuition fee loan and maintenance loan.
+
+[Find out if you're eligible for a bursary or scholarship](/funding-and-support/scholarships-and-bursaries).
+
+## If you're from other UK countries or Crown Dependencies
```

Comment:
@emmaachesongray I'm not sure `other UK countries` quite makes sense in this heading as we haven't mentioned England at all on this page - does this need changing?

```suggestion
## If you're from Wales, Scotland, Northern Ireland or the Crown Dependencies
```